.env.example

```diff
@@ -26,4 +26,11 @@ DISCORD_CHANNEL_UPDATE=
 
 # The website address used to access this epochtal instance. Please make sure this URL matches the external URL of your epochtal deployment.
 # Excluding protocol ("https://"), but including port if non-standard. For example: "epochtal.p2r3.com" or "epochtal.example.com:20000"
-#WEB_URL=
+# If left unset, defaults to "localhost:8080"
+WEB_URL=
+# Whether to host the webserver with TLS (requires a certificate)
+# If left unset, defaults to "false"
+USE_TLS=
```

Comment:
We should probably also tell users that they need to provide a certificate in `/app/secrets` if enabling this. I have no idea what the cleanest way to approach this would be - stating here that it needs to be mounted? Adding a comment to `docker-compose.yml`?